fiftyone/zoo/datasets/base.py

```
@@ -3461,11 +3908,30 @@ def _download_and_prepare(self, dataset_dir, scratch_dir, split):
     "quickstart-groups": QuickstartGroupsDataset,
     "quickstart-3d": Quickstart3DDataset,
+    "robomind": RoboMINDDataset,
     "sama-coco": SamaCOCODataset,
+    "sew-multimodal-amr": SEWMultimodalAMRDataset,
+    "tartanground": TartanGroundDataset,
+    "treescope-multimodal": TreeScopeMultimodalDataset,
     "ucf101": UCF101Dataset,
 }
 
 
+def _download_hub_dataset(repo_id, dataset_dir, revision=None):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_download_hub_dataset`

Add return type annotation: `None`

(ANN202)